vpr/src/device/rr_graph_view.h

```diff
@@ -53,10 +53,14 @@ class RRGraphView {
      */
   public:
     /* Get the type of a routing resource node */
-    t_rr_type node_type(RRNodeId node) const;
+    inline t_rr_type node_type(RRNodeId node) const {
```

Comment:
Should we comment why inline is used here? e.g. for runtime optimization etc. This can help other developers to understand and avoid them to remove inline property randomly.

Reply:
Yes, that makes sense. I'll add that.